drivers/gpio/gpio_pca953x.c

```diff
@@ -0,0 +1,512 @@
+/*
+ * Copyright (c) 2018 Peter Bigot Consulting, LLC
+ * Copyright (c) 2018 Aapo Vienamo
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ * Copyright (c) 2019 Vestas Wind Systems A/S
+ * Copyright (c) 2020 ZedBlox Ltd.
+ * Copyright (c) 2021 Laird Connectivity
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
Wow, the magic of open source is really showing itself in this list :smile:

Reply:
👍